src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -210,12 +221,14 @@ const constructPageFetchUrl = ({
         fetchUrl = Url(`/${service}${variantPath}/topics/${id}`);
         break;
       }
+      // in LIVE_PAGE case pathname is the ID of the Live page
+      // without /service/live/, and supports both Tipo & CPS IDs
```

Comment:
Let's move this comment to above line 230, and then we can remove the LIVE_PAGE case wording
```suggestion
      // pathname is the ID of the Live page without /service/live/, and supports both Tipo & CPS IDs
```